### PR TITLE
Gj 2392 hiding variables

### PR DESCRIPTION
**Pull request by emanuelemaffeo**

### CODE REVIEW

File: src/main/scala/com/getjenny/starchat/services/ResponseService.scala [hunk 1]
```diff
@@ -256,15 +257,15 @@ object ResponseService extends AbstractDataService {
           val analyzer: String = item.analyzer.declaration
           val stateData: Map[String, String] = item.stateData
 
-          val merged = searchResAnalyzers.extractedVariables ++ evaluationRes.data.extractedVariables
+          val mergedVariables = searchResAnalyzers.extractedVariables ++ evaluationRes.data.extractedVariables
           val randomizedBubbleValue = randomizeBubble(item.bubble)
-          val bubble = replaceTemplates(randomizedBubbleValue, merged)
-          val action = replaceTemplates(item.action, merged) //FIXME: action shouldn't contain templates
+          val bubble = replaceTemplates(randomizedBubbleValue, mergedVariables)
+          val action = replaceTemplates(item.action, mergedVariables) //FIXME: action shouldn't contain templates
```
Comment (angleto) [inline src/main/scala/com/getjenny/starchat/services/ResponseService.scala:263]: Codacy found an issue: [Take the required action to fix the issue indicated by this comment.](https://app.codacy.com/gh/GetJenny/starchat/pullRequest?prid=5736641)

**Comment by angleto** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           

Coverage increased per file
===========================
+ src/main/scala/com/getjenny/starchat/services/AnalyzerService.scala  1
+ src/main/scala/com/getjenny/starchat/services/ResponseService.scala  2
+ src/main/scala/com/getjenny/starchat/services/CronReloadDTService.scala  15
+ src/main/scala/com/getjenny/starchat/analyzer/atoms/http/custom/ZendeskSearchTicketsVariableManager.scala  66
         

Clones added
============
- src/test/scala/com/getjenny/starchat/resources/DecisionTableResourceTest.scala  17
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/GetJenny/starchat/pullRequest?prid=5736641&bid=18765628)

File: src/test/scala/com/getjenny/starchat/resources/DecisionTableResourceTest.scala [hunk 2]
```diff
@@ -575,6 +575,57 @@ class DecisionTableResourceTest extends TestEnglishBase {
...
+        evaluationClass = None,
+        maxResults = Some(1),
+        searchAlgorithm = Some(SearchAlgorithm.NGRAM3)
+      )
+
+      Post("/index_getjenny_english_0/get_next_response", request) ~> addCredentials(testUserCredentials) ~> routes ~> check {
+        status shouldEqual StatusCodes.OK
+        val response = responseAs[List[ResponseRequestOut]]
+        val headResponseRequestOut: ResponseRequestOut = response.headOption.getOrElse(fail)
+        headResponseRequestOut.actionInput.head shouldEqual Map("test template " -> "test action ").toJson.asJsObject
```
Comment (angleto) [inline src/test/scala/com/getjenny/starchat/resources/DecisionTableResourceTest.scala:624]: Codacy found an issue: [Usage of head on collections.](https://app.codacy.com/gh/GetJenny/starchat/pullRequest?prid=5736641)

File: build.sbt [hunk 3]
```diff
@@ -17,7 +17,7 @@ libraryDependencies ++= {
   val AnalyzerVersion = "3.0.6"
   val BreezeVersion	= "1.0"
   val CourierVersion = "1.0.0"
-  val ESClientVersion	= "7.5.1"
+  val ESClientVersion	= "7.6.0"
```
Comment (angleto) [inline build.sbt:20]: the version must not be changed

File: docker-starchat/docker-compose.double_sc.yml [hunk 4]
```diff
@@ -2,7 +2,7 @@ version: "3.2"
 services:
 
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:7.5.1
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.6.0
```
Comment (angleto) [inline docker-starchat/docker-compose.double_sc.yml:5]: the version must not be changed

File: docker-starchat/docker-compose.test.yml [hunk 5]
```diff
@@ -2,7 +2,7 @@ version: "3.2"
 services:
 
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:7.5.1
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.6.0
```
Comment (angleto) [inline docker-starchat/docker-compose.test.yml:5]: the version must not be changed

File: docker-starchat/docker-compose.yml [hunk 6]
```diff
@@ -2,7 +2,7 @@ version: "3.4"
 services:
 
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:7.5.1
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.6.0
```
Comment (angleto) [inline docker-starchat/docker-compose.yml:5]: the version must not be changed

File: docker-starchat/kibana.yml [hunk 7]
```diff
@@ -2,7 +2,7 @@ version: "3.4"
 services:
 
   kibana:
-    image: docker.elastic.co/kibana/kibana:7.5.1
+    image: docker.elastic.co/kibana/kibana:7.6.0
```
Comment (angleto) [inline docker-starchat/kibana.yml:5]: the version must not be changed

File: build.sbt (hunk 3 above)
Comment (emanuelemaffeo) [reply build.sbt:20]: It's strange because in the branch the version is 7.6.0...

Comment (angleto) [reply build.sbt:20]: The version on the 5.3 must be the 7.5.1 not 7.6.0

Comment (emanuelemaffeo) [reply build.sbt:20]: ok, maybe I've created the branch from the wrong version, closing this pr and creating a new one from a new branch

**Pull request closed without merging**